### PR TITLE
feat : landing page

### PR DESCRIPTION
**Pull request by ayush65**
Screenshot (58)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Dongle&family=Open+Sans:ital@1&family=Source+Sans+Pro:ital,wght@0,700;1,400&display=swap');
+@import url("./color-palette/color.css");
+
+body{
+    margin:auto !important;
```
Comment (NtshVrm) [inline style.css]: Not recommended to use !important tags

**Review by NtshVrm — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+.main-heading-content {
+    position: relative;
+    top: 20px;
+    color: var(--primary-color);
+}
+
+
+
```
Comment (shraddha15-creator) [inline style.css]: remove empty lines

**Review by shraddha15-creator — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (ayush65) [reply style.css]: done thanks for reviewing

File: style.css (hunk 2 above)
Comment (ayush65) [reply style.css]: done thanks for reviewing

File: style.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+}
+.landingpage-contents > p {
+    position: relative;
+    padding:20px;
+    margin-left: 50px;
+    margin-right: auto;
+    font-weight: 600;
+    font-size:20px;
+    color: var(--primary-color);
+
```
Comment (narharikale) [inline style.css]: avoid using tag itself as selector , instead use classes

File: style.css [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Dongle&family=Open+Sans:ital@1&family=Source+Sans+Pro:ital,wght@0,700;1,400&display=swap');
+@import url("./color-palette/color.css");
+
+body{
+    margin:auto !important;
+    padding:auto !important;
```
Comment (manismk) [inline style.css]: Try to not use important in css

**Review by manismk — APPROVED** [review]
Looks good

File: style.css (hunk 3 above)
Comment (ayush65) [reply style.css]: I have changed that  thanks for reviewing

File: style.css (hunk 4 above)
Comment (ayush65) [reply style.css]: i have changed that thanks for reviewing

**Review by narharikale — APPROVED** [review]
seems good now 👍

**Review by shraddha15-creator — APPROVED** [review]

**Review by HarshithaKonduru214 — COMMENTED** [review]
Looks good to me.

**Pull request merged**